src/translate_c.zig

```diff
@@ -5245,6 +5245,14 @@ const MacroCtx = struct {
         return self.list[self.i].id;
     }
 
+    fn skip(self: *MacroCtx, c: *Context, expected_id: CToken.Id) ParseError!void {
```

Comment:
Since `expected_id` will always be a a valueless tag, you can declare its type as `std.meta.Tag(CToken.Id)` (so that it's just the enum) and then the if statement can be `if (next_id != expected_id) {`